src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/BatchExecutionTest.java

```diff
@@ -167,6 +230,61 @@ public void testExecuteBatch1() {
             fail(TestResource.getResource("R_executeBatchFailed") + ": " + e.getMessage());
         }
     }
+    
+    public void testExecuteBatch1UseBulkCopyAPI() {
```

Comment:
This applies to all the tests, don't you need to set `useBulkCopyForBatchInsert` property to `true`?

Reply:
Done.